src/main/java/com/bootcamp/demo/service/exception/ItemNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.bootcamp.demo.service.exception;
+
+public class ItemNotFoundException extends RuntimeException {
+    public ItemNotFoundException() {
```

Comment:
When creating exceptions, it is usually best to use custom message which will outline the cause of the failure. You can find some details about this here: https://www.baeldung.com/java-new-custom-exception

Reply:
In this specific scenario, do not expose a constructor allowing one to create an exception without any useful message.